profiles/kentik_snmp/mcafee/mcafee-web-gateway.yml

```diff
@@ -0,0 +1,250 @@
+# https://github.com/librenms/librenms/blob/master/mibs/mcafee/MCAFEE-MWG-MIB
+---
+extends:
+  - if-mib.yml
+  - system-mib.yml
+
+provider: kentik-web-gateway
+
+sysobjectid:
+  - 1.3.6.1.4.1.1230.2.7.1.1    # McAfee Web Gateway 7
+
+metrics:
+  # Current overall CPU usage in percent
+  - MIB: MCAFEE-MWG-MIB
+    symbol:
+      name: stCPULoad
+      OID: 1.3.6.1.4.1.1230.2.7.2.5.1.0
+      tag: CPU
+  # Virtual memory consumption of the main processes divided by RAM + half of SWAP space. Projected to 0-100.
+  - MIB: MCAFEE-MWG-MIB
+    symbol:
+      name: stMemConsumed
+      OID: 1.3.6.1.4.1.1230.2.7.2.5.15.0
+      tag: MemoryUtilization
+  # Number of infections detected by the McAfee Gateway Antimalware Engine
+  - MIB: MCAFEE-MWG-MIB
+    symbol:
+      name: stMalwareDetected
+      OID: 1.3.6.1.4.1.1230.2.7.2.1.2.0
+  # Number of connections that not have been blocked
```

Comment:
have not been blocked?

Reply:
lol; copying directly from the MIB, I'll correct 😄